src/aqua/chain/deal_status.aqua

```diff
@@ -41,10 +41,10 @@ func poll_deal_statuses_by_ids(spell_id: SpellId, api_endpoint: string, deal_ids
                 if deal_status.status == DEAL_STATUS_ACTIVE:
                     activate_deal(spell_id, deal_status.deal_id)
                 else:
-                    if deal_status.status == DEAL_STATUS_INACTIVE:
+                    if deal_status.status == DEAL_STATUS_NOT_ENOUGH_WORKERS:
                         deactivate_deal(spell_id, deal_status.deal_id)
                     else: 
-                        if deal_status.status == DEAL_STATUS_ENDED:
+                        if deal_status.status == DEAL_STATUS_ENDED || deal_status == DEAL_STATUS_INSUFFICIENT_FUNDS:
```

Comment:
Are you sure it's wise to remove deals each time there's not enough money? it happens quite often

Reply:
developer should add funds and rematch

Reply:
when the deal is exited, it can't be funded and rematched, no?

Reply:
if deal ended it can't be funded

Reply:
:ok_hand:

Reply:
We don't change deal state in case of exiting from provider side

Reply:
what is priority?

Reply:
I mean, is a deal has both not enough workers and insufficient funds, which status will be shown?

Reply:
insufficient

Reply:
should be, but it's not now, wow